### DISCUSSION TITLE
draco_decoder issue

### DISCUSSION THREAD

**Discussion by Kevin-Hamilton** [pmndrs/drei#575] [Q&A]
I'm suddenly getting RuntimeErrors in WebAssembly associated with my calls to useGLTF().

Looking at the source code, I saw I could pass a URL as the 2nd parameter of useGLTF, so I updated my calls to use the URL https://www.gstatic.com/draco/versioned/decoders/1.4.3/ and that seems to have fixed the problem.

It looks like the recommendation here: https://github.com/google/draco is to use that versioned URL format instead of the https://www.gstatic.com/draco/v1/decoders/ that drei is using here: https://github.com/pmndrs/drei/blob/c255ecb38e213a90190cbf6284ddac5a742dbb83/src/core/useGLTF.tsx#L13

It seems highly unusual that Google would introduce breaking changes to a previously stable URL like https://www.gstatic.com/draco/v1/decoders/  so I'm wondering if anyone else is seeing the same issue as me, or is it just me?

**Comment by oyvinvo** [comment]
We're experiencing the same issue after google released draco 1.4.3 yesterday.

**Comment by joshuaellis** [comment]
I think theres an issue open here – https://github.com/google/draco/issues/751

**Comment by CodyJasonBennett** [comment]
From Discord:

![image](https://user-images.githubusercontent.com/23324155/137455714-3cdd3cda-bf93-4015-b8f1-cc7e6903f26d.png)
